org.eclipse.scout.rt.server.commons/src/main/java/org/eclipse/scout/rt/server/commons/authentication/TrivialAccessController.java

```diff
@@ -254,4 +267,30 @@ public TrivialAuthConfig withLoginPageInstalled(final boolean loginPageInstalled
       return this;
     }
   }
+
+  /**
+   * Verify if the request is a browser /unload/ request. Unload requests are handled by {@code UnloadRequestHandler}.
+   */
+  protected boolean isUnloadRequest(final HttpServletRequest request) {
```

Comment:
this method could be public static and then be used in UnloadRequestHandler?
Furthermore, I would remove all these unnecessary "final" keywords in this class to remove noise.

Reply:
I disagree. I don't like it to have the logic "what is an unload request" in two places.
Its for sure wrong in the TrivialAccessController. But still it should be in one place.
Could be in a new class.

Reply:
Agreed! Lets find a good place for that logic. How about `ServletFilterHelper`?

Reply:
Yes, sounds good!